### ISSUE TITLE
AssertionError Output Spam When Running T5ForConditionalGeneration

### ISSUE THREAD

**Issue by eellison** [pytorch/torchdynamo#696]
Repro:
`python benchmarks/huggingface.py --float32 -dcuda --training --nvfuser --accuracy-aot-ts-mincut --use-eval-mode --isolate --only=T5ForConditionalGeneration`

Output:
```
ERROR FROM offset=4 filename /scratch/eellison/work/env/lib/python3.9/site-packages/apex/_autocast_utils.py 20 AssertionError
ERROR FROM offset=10 filename /scratch/eellison/work/env/lib/python3.9/site-packages/apex/normalization/fused_layer_norm.py 187 AssertionError
ERROR FROM offset=48 filename /scratch/eellison/work/env/lib/python3.9/site-packages/apex/normalization/fused_layer_norm.py 386 AssertionError
ERROR FROM offset=6 filename /scratch/eellison/work/env/lib/python3.9/site-packages/transformers/models/t5/modeling_t5.py 569 AssertionError
ERROR FROM offset=154 filename /scratch/eellison/work/env/lib/python3.9/site-packages/transformers/models/t5/modeling_t5.py 664 AssertionError
...
ERROR FROM offset=48 filename /scratch/eellison/work/env/lib/python3.9/site-packages/apex/normalization/fused_layer_norm.py 386 AssertionError
ERROR FROM offset=6 filename /scratch/eellison/work/env/lib/python3.9/site-packages/transformers/models/t5/modeling_t5.py 603 AssertionError
```

**Comment by mlazos** [comment]
Fixed by #793
